node-perf-dash/kubelet-parser.go

```diff
@@ -188,7 +194,7 @@ func GrabTracingKubelet(d Downloader, job string, buildNumber int, nodeName stri
 			"test": sortedTestTime[currentTestIndex].TestName,
 			"node": nodeName,
 		},
-		Version: tracingVersion,
+		Version: perftype.CurrentKubeletPerfMetricsVersion,
```

Comment:
I don't understand, why do we add version on the node-perf-dash side?

The version is used to identify whether the version of the original data, it doesn't seem to be the original purpose of the versioning stuff.